package.json

```
@@ -32,7 +32,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "microformats-parser": "^1.4.0",
+    "microformats-parser": "^1.5.1",
```

```suggestion
    "microformats-parser": "^2.0.0",
```

Might be an idea to pick the new major version